load_db/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM ubuntu:16.04
+FROM python:3.6-jessie
```

Comment:
This causes some errors on the second import, because an older version of ogr2ogr is installed.  
I would use `python:3.6-stretch` (to install [`gdal-bin`](https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=gdal-bin) 2.1.2 instead of 1.10.1).